Library/Homebrew/brew.sh

```diff
@@ -172,6 +172,11 @@ case "$@" in
     source "${HOMEBREW_LIBRARY}/Homebrew/list.sh"
     homebrew-list "$@" && exit 0
     ;;
+  # falls back to cmd/tap.rb on a non-zero return
+  tap*)
```

Comment:
```suggestion
  tap)
```
I wonder if this makes more sense so we don't have to handle `tap*` in `homebrew-tap`